pygef/grouping.py

```diff
@@ -69,7 +69,7 @@ def group_significant_layers(df_group, min_thickness, start_depth):
             "zf": df_group.zf,
         }
     )
-    df_group["zf"].iloc[-1] = depth
+    df_group.iloc[-1, 2] = depth
```

Comment:
maybe it's better: `df_group.loc[df_group.shape[0]-1, "zf"] = depth`

In this way we keep it independent from the position of `"zf" `

Reply:
Good suggestion. Perhaps this is the most pythonic way of doing it:
`df_group.iloc[-1, df_group.columns.get_loc("zf")] = depth`